### DISCUSSION TITLE
渐变色在坐标轴变换后显示效果也期望变换。

### DISCUSSION THREAD

**Discussion by ChristmasFox** [antvis/G2#5448] [Q&A]
由于连续型数据不好用style.fill的linear-gradint体现数据大小的变化，所以没法用它画纵向渐变,
而使用'gradient': 'y'又有局限性：
1、不能展示color通道的渐变变化，只能体现y通道的
2、坐标轴变换后不能变为纵向

<img width="1095" alt="image" src="https://github.com/antvis/G2/assets/68337739/52961d2e-773f-41fe-904a-83e4e4eae6a9">
坐标轴变换后，渐变色仍然为横向渐变，期望能纵向渐变
<img width="970" alt="image" src="https://github.com/antvis/G2/assets/68337739/ddb4853b-1919-48b5-aaec-d5c5662b1127">

**Comment by pearmini** [comment]
- 相同的问题：https://github.com/antvis/G2/discussions/5333
- 可以跟进这个 issue：https://github.com/antvis/G2/issues/5361

**Reply by ChristmasFox** [reply]
好的 是正在修复这个问题吗 [#5361](https://github.com/antvis/G2/issues/5361)
